helm_deploy/hmpps-registers-to-delius-update/templates/_envs.tpl

```diff
@@ -1,85 +0,0 @@
-    {{/* vim: set filetype=mustache: */}}
-{{/*
-Environment variables for web and worker containers
-*/}}
-{{- define "deployment.envs" }}
-env:
-  - name: SERVER_PORT
-    value: "{{ .Values.image.port }}"
-
-  - name: JAVA_OPTS
-    value: "{{ .Values.env.JAVA_OPTS }}"
-
-  - name: OAUTH_ENDPOINT_URL
-    value: "{{ .Values.env.OAUTH_ENDPOINT_URL }}"
-
-  - name: PROBATION_ENDPOINT_URL
-    value: "{{ .Values.env.PROBATION_ENDPOINT_URL }}"
-
-  - name: COURT_REGISTER_ENDPOINT_URL
-    value: "{{ .Values.env.COURT_REGISTER_ENDPOINT_URL }}"
-
-  - name: SPRING_PROFILES_ACTIVE
-    value: "logstash"
-
-  - name: APPINSIGHTS_INSTRUMENTATIONKEY
-    valueFrom:
-      secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: APPINSIGHTS_INSTRUMENTATIONKEY
-
-  - name: APPLICATIONINSIGHTS_CONNECTION_STRING
-    value: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
-
-  - name: APPLICATIONINSIGHTS_CONFIGURATION_FILE
-    value: "{{ .Values.env.APPLICATIONINSIGHTS_CONFIGURATION_FILE }}"
-
-  - name: HMPPS_AUTH_CLIENT_CLIENT_ID
```

Comment:
these two need to be copied from the secret in `values.yml`

Reply:
APPLICATIONINSIGHTS_CONFIGURATION_FILE set in individual files.  Added HMPPS_AUTH_CLIENT_CLIENT_ID